### ISSUE TITLE
Pobieranie dodatkowych metryk aby móc lepiej rozumieć jak wygląda problem.

### ISSUE THREAD

**Issue by DocentSzachista** [DocentSzachista/MGU#7]
Na razie mamy accuracy i loss w sieci na datasecie testowym, lecz w zbiorach niezbalansowanych te metryki nie są wystarczające. Aby lepiej zrozumieć problem dobrze by było móc poszerzyć statystykę o to ile obrazków ze zbioru testowego rozpoznał poprawnie. 
Moja propozycja to kiedy będziemy testować sieci, to wydobywać wszystkie logits'y z sieci dla każdego testowego obrakza i zapisywać do dataframe'a pandasowego, wraz z informacją jak był oznaczony obrazek. Następnie będzie można wyznaczać na podstawie tego takie rzeczy jak:
**confusion matrix**, **precision**, czy też **recall**. Przykłady tych metryk oraz innych [podane tutaj](https://www.kaggle.com/code/marcinrutecki/best-techniques-and-metrics-for-imbalanced-dataset#14.-Other-Metrics)

**Comment by DocentSzachista** [comment]
Przykład implementacji confussion matrix https://christianbernecker.medium.com/how-to-create-a-confusion-matrix-in-pytorch-38d06a7f04b7
